### ISSUE TITLE
Update broke some tables

### ISSUE THREAD

**Issue by xLink** [yajra/laravel-datatables#100]
just ran a composer update, updated from `v5.4.5` (where the tables in question worked) to latest, `v5.5.3` and the table stopped working

Error:
>  ErrorException thrown with message "array_keys() expects parameter 1 to be array, object given"
> 
> Stacktrace:
> #82 ErrorException in ./project/vendor/yajra/laravel-datatables-oracle/src/yajra/Datatables/Engines/CollectionEngine.php:46

While the data is being manually added to a Collection, the datatable is getting a `Illuminate\Support\Collection` object.

Should probably be noted my other datatables that use Eloquent, are still working ok.

```php
Illuminate\Support\Collection {#1198
  #items: array:4 [
    2 => {#1163
      +"order": 0
      +"name": "Core"
      +"alias": "core"
      +"authors": array:1 [
        0 => {#1152
          +"name": "xLink (Dan Aldridge)"
          +"email": "xLink@cybershade.org"
        }
      ]
      +"version": "v1.0.0"
      +"keywords": array:1 [
        0 => "core-module"
      ]
      +"active": true
      +"path": "./project/app/Modules/Core"
    }
    1 => {#1177
      +"order": 1
      +"name": "Auth"
      +"alias": "auth"
      +"authors": array:1 [
        0 => {#1167
          +"name": "xLink (Dan Aldridge)"
          +"email": "xLink@cybershade.org"
        }
      ]
      +"version": "v1.0.0"
      +"keywords": array:1 [
        0 => "core-module"
      ]
      +"active": true
      +"path": "./project/app/Modules/Auth"
    }
    0 => {#1178
      +"order": 2
      +"name": "Admin"
      +"alias": "admin"
      +"authors": array:1 [
        0 => {#1188
          +"name": "xLink (Dan Aldridge)"
          +"email": "xLink@cybershade.org"
        }
      ]
      +"version": "v1.0.0"
      +"keywords": array:1 [
        0 => "core-module"
      ]
      +"active": true
      +"path": "./project/app/Modules/Admin"
    }
}
```

**Comment by yajra** [comment]
@xLink, sorry for the inconvenience but I cannot reproduce the error? I just update my demo site and collection engine seems to work fine. http://datatables.yajrabox.com/collection/basic-object. Can you please provide more info to replicate the issue?

**Comment by xLink** [comment]
Okay after further digging looks like it doesnt like the fact each element in the collection is a `stdObject`, might be worth adding an example where by you manually build a Collection from an array

**Comment by yajra** [comment]
Ok, I diff the v5.4.5 to latest and the diff is below. Can you please test the code if it will fix the issue? On line 43 update it to:

```php
$this->columns             = array_keys($this->serialize((array) $collection->first()));
```

**Comment by xLink** [comment]
yeah just tested it on my broken table, that seems to have fixed it

**Comment by yajra** [comment]
Thanks for confirming. Should be fixed on v5.5.4
